src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-pop-up/ubs-admin-tariffs-location-pop-up.component.ts

```diff
@@ -72,8 +72,8 @@ export class UbsAdminTariffsLocationPopUpComponent implements OnInit, AfterViewC
   autocompleteLsr;
   name: string;
   unsubscribe: Subject<any> = new Subject();
-  datePipe = new DatePipe('ua');
-  newDate;
+  datePipe = new DatePipe(this.languageService.getCurrentLanguage());
```

Comment:
You can subscribe to `localStorageService.languageBehaviouralSubject` to get language and then use it as parameter for pipe in template

Reply:
Okay, thank you, added this in new commit